recipes/ligotimegps/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ligotimegps" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b3a53a7ec5728e8510f8a1a48e833c6ab9dbf7ee3b15aeef03790917de6241f7
+
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```

Comment:
Shouldn't this be host?

Reply:
@scopatz, I'm not really sure when to use `build` and `host`, my vague understanding is that if a dependency is only required for building, and does not need to be present at run time, use `build`, otherwise use `host` if it is required at runtime (for linking).

Since `pip` and `setuptools` here are only required to build the tarball, and are not required to use `ligotimegps`, is `build` not the right location? I'm happy to be educated on the nuances of this.

Reply:
Yeah, unfortunately, this is something that we are all getting used to in the advent of conda-build 3 (cc @msarahan). The docs have gotten better on this recently, though. See: https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#requirements-section 

Basically, `build` is for the packages that need to be on the machine that is creating the package, ie make and compilers.  `host` is for the packages that are needed on the target platform.  This is definitely opposite of my intuition (where host would be for packages that need to be on the  host system, and build is for the target build), but there is no going back at this point :wink:

Reply:
@scopatz, thanks, I think them that `build` is the correct place, since `pip` and `setuptools` are required to run the `python -m pip install` command that builds the package tarball (which runs on the build machine). `pip` is not required to _use_ the installed package.

Reply:
pip and setuptools should be in the `host` section as should python.  From the [conda docs](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host):

---

Host ...
You should also include the base interpreter for packages that need one. In other words, a Python package would list python here and an R package would list mro-base or r-base.

---

In this case because there is not a build section the recipes will be fine with these requirements in the build section but a better place to put them is in host.  

The `build` and `host` names follow automake's [naming for cross cross-compilation](https://www.gnu.org/software/automake/manual/html_node/Cross_002dCompilation.html).  Projects which use automake accept `--build` and `--host` arguments to `./configure`